tip.lic

```diff
@@ -0,0 +1,99 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#tip
+=end
+
+# Background: I've always been annoyed by the need to specify tips in coppers.
```

Comment:
This is the kind of info that would normally reside on elanthipedia in the docs for this script. Not opposed to keeping it here if you think it needs to stay, just pointing it out.

Reply:
No problem. I can put it on the -pedia.